certified-connectors/Azure Communication Services Identity/apiProperties.json

```diff
@@ -0,0 +1,29 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Connection String",
+          "description": "Azure Communication Services Connection String.",
+          "tooltip": "Find your connection string in your Azure Communication Services resource in Azure Portal in \"Keys\".",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@lei-msft this color is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector). Can you please change it?